troubleshoot/ingest/opentelemetry/edot-sdks/java/proxy-issues.md

```diff
@@ -0,0 +1,58 @@
+---
+navigation_title: Proxy issues
+description: Troubleshooting guide for the EDOT Java SDK, covering proxy issues.
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    edot_java: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+---
+
+# Proxy issues with EDOT Java SDK
+
+If your Java SDK sends telemetry but fails to communicate with the APM server, the issue might be due to missing or misconfigured proxy settings, which are required for outbound HTTP/S communication in some environments.
+
+## Symptoms
+
+These symptoms typically indicate that your Java SDK is experiencing a proxy issue:
+
+* Telemetry data is not received in Elastic APM after instrumenting the app.
+* Logs show errors like `Connection timed out`, `TLS handshake failure`, or other network-related issues when connecting to Elastic APM.
+
+## Resolution
+
+Set standard Java system properties to configure proxy settings. 
+
+Configure the following JVM system properties to enable outbound HTTP/HTTPS communication through a proxy:
+
+```bash
+java \
+-Dhttp.proxyHost=proxy.example.com \
+-Dhttp.proxyPort=8080 \
+-Dhttps.proxyHost=proxy.example.com \
+-Dhttps.proxyPort=8080 \
```

Comment:
I did expect this to not work, because to my knowledge the exporters use `OkHttp3`, which I thought would not respect these properties. However, according to [this issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/12240) it should work. Did we test this?